src/assets/button.module.css

```diff
@@ -0,0 +1,31 @@
+.Button {
+  width: 25%;
+  height: 100%;
+  font-size: 3rem;
+  font-family: inherit;
+  background: #cad2c5;
+  border: 1px solid transparent;

```

Comment:
- [ ] To fit with the original design requirement, kindly avoid the `transparent` property of the buttons. It needs a visible border as you can compare it with the original design. 
- [ ] The button text looks a bit extra big. It doesn't achieve a good size contrast and affects the user experience. Kindly, follow the original design. 
- [ ] OPTIONAL: While it still looks ok, you can still make changes the background colors to meet the original design requirements. 
![image](https://user-images.githubusercontent.com/64894927/129790737-d5eab408-819e-4e59-94ef-b8a7e68781c4.png)
![image](https://user-images.githubusercontent.com/64894927/129790798-e20c4b63-71ed-460a-8c2a-4ef8d7e13839.png)

Reply:
Resolved